src/data_gradients/managers/detection_manager.py

```diff
@@ -45,6 +54,7 @@ def __init__(
         :param train_data:              Iterable object contains images and labels of the training dataset
         :param val_data:                Iterable object contains images and labels of the validation dataset
         :param config_path:             Full path the hydra configuration file. If None, the default configuration will be used.
+        :param feature_extractors:      One or more feature extractors to use. If None, the default configuration will be used.
```

Comment:
I would add at the end
- "Mutually exclusive with `feature_extractors`."
- "Mutually exclusive with `config_path`."

Reply:
added